code/src/index.css

```diff
@@ -1,13 +1,252 @@
+html {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+}
+
+h4 {
+  font-size: 1rem;
+}
+p {
+  font-size: 1rem;
+}
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: linear-gradient(to top, #ebbba7 0%, #cfc7f8 100%);
+  background-attachment: fixed;
+  display: flex;
+  justify-content: center;
+}
+
+#root {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.post-container {
+  width: 93%;
+  background-color: rgb(242, 240, 240);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0.71rem 0.71rem;
+  border: solid 1px black;
+  box-shadow: 5px 5px black;
+}
+.post-container-message {
+  width: 93%;
+  height: 100%;
+  background-color: white;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0.71rem 0.71rem;
+  border: solid 1px black;
+  box-shadow: 5px 5px black;
+}
+.title {
+  width: 90%;
+  text-align: left;
+}
+
+.text-container {
+  width: 90%;
+  height: 5rem;
+  background-color: white;
+  border: solid 1px black;
+}
+.text-area {
+  width: 95%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.text-area p {
+  margin: 0;
+  text-align: left;
+}
+.message-container {
+  width: 90%;
+  height: 4.8rem;
+  padding: 1rem 0px;
+}
+.message-container p {
+  margin: 0;
+  margin-top: 5px;
+}
+button {
+  height: 2.28rem;
+  background-color: #ffadad;
+  border-radius: 0.93rem;
+  border-color: #ffadad;
+}
+.send-button {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  height: 1rem;
+  text-align: left;
+  padding: 3rem 0px;
+}
+.likes {
+  display: flex;
+  justify-content: space-between;
+}
+.likes p {
+  padding: 0rem 0.71rem;
+}
+.time {
+  text-align: end;
+}
+.send-button img {
+  width: 2.2rem;
+  height: 2.2rem;
+  background-color: #ffadad;
+  border-radius: 50%;
+}
+.length-error {
+  color: red;
+  font-weight: bolder;
+}
+.error-message {
+  width: 90%;
+}
+h3 {
+  text-align: justify;
+  padding: 0px 08px;
+}
+.label {
+  height: 29px;
+  background-color: #ffadad;
+  width: 128px;
+  text-align: center;
+  border-radius: 10px;
+  color: #8f2525;
+  font-weight: bolder;
+  box-shadow: 2px 2px;
+}
+
+@media (min-width: 768px) {
```

Comment:
Site looks good at this screen size and larger, but phone size is a bit unresponsive. Consider going Mobile First for future projects, as that is the most common browsing platform these days

Reply:
I started mobile-first but now,  I am struggling with a huge problem in my computer with the screens, after finished the project in mobile version and changed between devices all look good but, after uploading the site to netlify the behavior is totally different I don't know why but basically I need to resize all the style.